add-on/chrome/bootstrap.js

```diff
@@ -596,11 +590,9 @@ var WindowListener = {
       /**
        * Hides the infobar, permanantly if requested.
        *
-       * @param {Boolean} permanently Flag that determines if the infobar will never
-       *                              been shown again. Defaults to `false`.
        * @return {Boolean} |true| if the infobar was hidden here.
        */
-      _hideBrowserSharingInfoBar: function(permanently = false, browser) {
+      _hideBrowserSharingInfoBar: function(browser) {
```

Comment:
nit: could you add a `@param`  comment for browser indicating it's optional and it's for the content browser to remove infobar or defaulting to current browser.